sim_benchmark/robot_df/turtlebot/model.urdf

```diff
@@ -0,0 +1,750 @@
+<?xml version="1.0" ?>
+<!-- =================================================================================== -->
+<!-- |    This document was autogenerated by xacro from robots/kobuki_hexagons_kinect.urdf.xacro | -->
```

Comment:
Please use the complete url for the original file.
Where did you get this file? Is there any copyright/license information we need to include?

Reply:
I'm sorry to put the model file information. 

The file was auto generated by a script [xacro](http://ros.org/wiki/xacro). Original file is kobuki_hexagons_kinect.urdf.xacro, which comes from ros package [turtlebot_description](http://wiki.ros.org/turtlebot_description).  

The file can be found here:
https://github.com/turtlebot/turtlebot/blob/kinetic/turtlebot_description/robots/kobuki_hexagons_kinect.urdf.xacro

I removed some sensors so that it can be compatible with pybullet. I'm not sure what license information should we include here?

Reply:
You can include the above information in the file and describe what changes you make to the file. Also include the LICENSE file in the original repo: https://github.com/turtlebot/turtlebot/blob/kinetic/LICENSE

Reply:
LICENSE information and changes added.